sdk/core/core-tracing/test/interfaces.spec.ts

```diff
@@ -30,7 +30,7 @@ describe("interface compatibility", () => {
       parent: context
     };
 
-    const OTSpanOptions: OTSpanOptions = spanOptions;
-    assert.ok(OTSpanOptions);
+    const OTOptions: OTSpanOptions = spanOptions;
+    assert.ok(OTOptions);
```

Comment:
what was the linting error?

Reply:
@deyaaeldeen there was originally a no-unused-vars lint error on line 6, where we import (OTSpanOptions) https://eslint.org/docs/rules/no-unused-vars

Renaming the constant where we specify the type as OTSpanOptions fixed the issue.

Reply:
Interesting, so the rule thought that the variable name shadowed the type name. That sounds like a bug to me. I did some investigation and found the following plugin: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-shadow.md. Would you please test it with the option `ignoreTypeValueShadow` enabled to see if this solves the issue without the code change in this PR?

Reply:
@deyaaeldeen I tried fixing it doing that, but the lint issue still pops up (see below).

![image](https://user-images.githubusercontent.com/8355075/97997557-1399cf80-1db7-11eb-96d8-9486489c5cb4.png)
![image](https://user-images.githubusercontent.com/8355075/97997636-2ad8bd00-1db7-11eb-8097-95184dbd3c79.png)

Reply:
@abc516 thanks for the try! However I meant the `rules` key to be set as follows:
```
...
  "rules": {
    "no-shadow": "off",
    "@typescript-eslint/no-shadow": ["error", { ignoreTypeValueShadow: true }]
  }
...
```